onboarding-enabler-spring/src/main/java/org/zowe/apiml/enable/config/EnableApiDiscoveryConfig.java

```diff
@@ -72,7 +72,7 @@ public ApiMediationClient apiMediationClient(EurekaClientProvider eurekaClientPr
         }
     }
 
-    @ConfigurationProperties(prefix = "apiml.service")
+    @ConfigurationProperties(prefix = "apiml.service", ignoreInvalidFields = true)
```

Comment:
Why did you decide for this ignore?

Reply:
this will allow to have empty values in configuration what will be the case for SSL in AT-TLS

Reply:
ok